base/networking.c

```diff
@@ -782,9 +782,9 @@ ip_islocalhost (struct sockaddr_storage *storage)
   struct in_addr addr;
   struct in_addr *addr_p;
   struct in6_addr addr6;
-  struct in6_addr *addr6_p;
+  struct in6_addr *addr6_p = NULL;
```

Comment:
These look wrong, because they're set directly below.  Wouldn't recommend adding them just to quiet clang, but if you want to I recommend adding a note that it's for clang.